README.md

```diff
@@ -1,9 +1,7 @@
 rmtrash 1.14
 ============
 
-Put files (and directories) in trash using the ```trash-put``` command in a way that is, otherwise as ```trash-put``` itself, compatible to GNUs ```rm``` and ```rmdir```.
-
-[Click here for more information about trash-cli.](https://github.com/andreafrancia/trash-cli)
+Put files (and directories) in trash using the ```trash-put``` command from [trash-cli](https://github.com/andreafrancia/trash-cli), in a way that is (as ```trash-put``` itself) compatible to GNUs ```rm``` and ```rmdir```.
```

Comment:
Doesn't "in a way that is (as ```trash-put``` itself) compatible to GNUs ```rm``` and ```rmdir```" imply that `trash-put` is compatible with `rm` and `rmdir`?

Reply:
It does — that's what I interpreted the original phrasing as saying 🤔. So you meant the opposite? Perhaps this would be clearer, then:

> in a way that (unlike ```trash-put``` itself) is compatible to GNUs ```rm``` and ```rmdir```.

Reply:
That would be perfect :+1:

Yeah, `trash-put` isn't really compatible with `rm` resp. `rmdir`. It is API compatible, i.e. it won't fail when passing options of `rm` and/or `rmdir`, but the options have no effect. `trash-put` behaves very different. The purpose of `rmtrash` is to mimic `rm`'s and `rmdir`'s behaviour.

Reply:
Great, thanks for confirming! I've updated the file accordingly.